.github/workflows/release.yml

```diff
@@ -23,17 +27,23 @@ jobs:
           with:
             java-version: 11
         - name: Decode Keystore
-        - run: |
+          run: |
             echo "${{secrets.RELEASE_KEYSTORE}}" > release.keystore.asc
             gpg -d -passphrase "${{secrets.RELEASE_KEYSTORE_PGP_PASSPHRASE}}" -batch release.keystore.asc > app/release.keystore
         - name: Generate release build
           run: ./gradlew assemble
         - name: Execute exodus-standalone
-          uses: docker://exodusprivacy/exodus-standalone:v1.3.1
+          uses: docker://exodusprivacy/exodus-standalone:v1.3.2
           with:
             args: /github/workspace/app/build/outputs/apk/release/app-release-signed.apk
         - name: Upload APK
           uses: actions/upload-artifact@v3
           with:
             name: app_release
             path: app/build/outputs/apk/release/app-release-signed.apk
+        - name: Create release
+          if: github.event_name == 'push' && github.ref_type == 'tag'
```

Comment:
```suggestion
```

Not necessary anymore as the condition is on the whole file

Reply:
I have add this to allow run workflow dispatch without create release on Github.
I can build apk release only to submit on Play Store and after Google approved create tag to push on Fdroid.